blobfuse2-release.yaml

```diff
@@ -429,6 +429,7 @@ stages:
               chmod 777 *.sh
               ./dockerinstall.sh
               ./buildcontainer.sh
+              ./buildcontainer_alpine.sh
```

Comment:
With this change, we will be building for both ubuntu and alpine based docker images, right?